examples/misc/zorder_demo.py

```diff
@@ -3,66 +3,69 @@
 Zorder Demo
 ===========
 
-The default drawing order for axes is patches, lines, text.  This
-order is determined by the zorder attribute.  The following defaults
-are set
+The drawing order of artists is determined by their ``zorder`` attribute, which
+is a floating point number. Artists with higher ``zorder`` are drawn on top.
+You can change the order for individual artists by setting their ``zorder``.
+The default value depends on the type of the Artist:
 
-=======================    =======
-Artist                     Z-order
-=======================    =======
-Patch / PatchCollection    1
-Line2D / LineCollection    2
-Text                       3
-=======================    =======
+================================================================    =======
+Artist                                                              Z-order
+================================================================    =======
+Images (`.AxesImage`, `.FigureImage`, `.BboxImage`)                 0
+`.Patch`, `.PatchCollection`                                        1
+`.Line2D`, `.LineCollection` (including minor ticks, grid lines)    2
+Major ticks                                                         2.01
+`.Text` (including axes labels and titles)                          3
+`.Legend`                                                           5
+================================================================    =======
 
-You can change the order for individual artists by setting the zorder.  Any
-individual plot() call can set a value for the zorder of that particular item.
+Any individual `~.axes.Axes.plot()` call can set a value for the zorder of that
```

Comment:
really that should be "any call to a plotting method" (in general, not just plot()).